step-templates/octopus-chain-deployment.json

```diff
@@ -3,13 +3,13 @@
ForDeployment) {\n    Write-Host 'Deployments have been queued, proceeding to the next step...'\n    return\n}\n\nShow-Heading 'Waiting For Deployment'\ndo {\n    Start-Sleep -Seconds 1\n    $tasksStillRunning = $false\n    foreach ($deployment in $deploymentControllers) {\n        if ($deployment.PollCheck()) {\n            $tasksStillRunning = $true\n        }\n    }\n} while ($tasksStillRunning)\n\nif ($deploymentControllers | ForEach-Object Task | Where-Object FinishedSuccessfully -eq $false) {\n    Show-Heading 'Deployment Failed!'\n    Write-Fatal (($deploymentControllers | ForEach-Object Task | ForEach-Object ErrorMessage) -join \"`n\")\n}\nelse {\n    Show-Heading 'Deployment Successful!'\n}\n\nif (Test-String $Chain_PostDeploy -ForAbsence) {\n    return \n}\n\nShow-Heading 'Post-Deploy Script'\n$rawPostDeployScript = Invoke-OctopusApi (\"$Chain_BaseApiUrl/releases/{0}\" -f $OctopusParameters['Octopus.Release.Id']) |\nForEach-Object { Invoke-OctopusApi (Format-LinksUri -Uri $_.Links.ProjectDeploymentProcessSnapshot) } |\nForEach-Object Steps | Where-Object Id -eq $OctopusParameters['Octopus.Step.Id'] |\nForEach-Object Actions | Where-Object Id -eq $OctopusParameters['Octopus.Action.Id'] |\nForEach-Object { $_.Properties.Chain_PostDeploy }\nWrite-Verbose \"Raw Post-Deploy Script:`n$rawPostDeployScript\"\n\nAdd-Type -Path (Get-WmiObject Win32_Process | Where-Object ProcessId -eq $PID | ForEach-Object { Get-Process -Id $_.ParentProcessId } | ForEach-Object { Join-Path (Split-Path -Path $_.Path -Parent) 'Octostache.dll' })\n\n$deploymentControllers | ForEach-Object {\n    $deployment = $_.Task.Deployment\n    $tenant = $_.Tenant\n    $variablesDictionary = [Octostache.VariableDictionary]::new()\n    Invoke-OctopusApi (\"$Chain_BaseApiUrl/variables/{0}\" -f $deployment.ManifestVariableSetId) | ForEach-Object Variables | Where-Object {\n        ($_.IsSensitive -eq $false) -and `\n        ($_.Scope.Private -ne 'True') -and `\n        ($null -eq $_.Scope.Action) -and `\n        ($null -eq $_.Scope.Machine) -and `\n        ($null -eq $_.Scope.TargetRole) -and `\n        ($null -eq $_.Scope.Role) -and `\n        ($null -eq $_.Scope.Tenant -or $_.Scope.Tenant -contains $tenant.Id) -and `\n        ($null -eq $_.Scope.TenantTag -or (Compare-Object $_.Scope.TenantTag $tenant.TenantTags -ExcludeDifferent -IncludeEqual)) -and `\n        ($null -eq $_.Scope.Environment -or $_.Scope.Environment -contains $deployment.EnvironmentId) -and `\n        ($null -eq $_.Scope.Channel -or $_.Scope.Channel -contains $deployment.ChannelId) -and `\n        ($null -eq $_.Scope.Project -or $_.Scope.Project -contains $deployment.ProjectId)\n    } | ForEach-Object { $variablesDictionary.Set($_.Name, $_.Value) }\n    $postDeployScript = $variablesDictionary.Evaluate($rawPostDeployScript)\n    Write-Host \"$($_.LogPrefix)Evaluated Post-Deploy Script:\"\n    Write-Host $postDeployScript\n    Write-Host 'Script output:'\n    [scriptblock]::Create($postDeployScript).Invoke()\n}"
```

Comment:
On line 98, you've added the following lines:

```powershell
$DefaultUrl = $OctopusParameters['Octopus.Web.BaseUrl']
$Chain_BaseApiUrl = "/api"
```
The second line (`$Chain_BaseApiUrl = "/api"`) was originally on line 235 and that's been removed :heavy_check_mark: 

But the first line (`$DefaultUrl = $OctopusParameters['Octopus.Web.BaseUrl']`) now exists on lines 98 and 138.

Can you please either:
- remove line 98 or line 138 (I think line 138 would make the most sense)
- revert the changes to both lines 98+99 and reinstate line 235

Reply:
Hi @harrisonmeister 
I updated the template using your input.
Thank you for the great feedback!